ampersand-form-view.js

```diff
@@ -164,11 +166,13 @@ module.exports = View.extend({
 
     render: function () {
         if (this.rendered) return;
-        if (!this.el) {
-            this.el = document.createElement('form');
-        }
+
+        this.renderWithTemplate(this);
+
+        this.formEl = this.query('form') || this.el;
+
         if (this.autoAppend) {
-            this.fieldContainerEl = this.el.querySelector('[data-hook~=field-container]') || this.el;
+            this.fieldContainerEl = this.queryByHook('field-container') || this.formEl;
```

Comment:
I took the liberty of using this ampersand view convenience method (`queryByHook`), but I apologize if there was a reason it wasn't used before.